tests/utils/fixtureUrl.js

```diff
@@ -0,0 +1,14 @@
+module.exports.getUrlFixture = ({
+	siteUrl,
+	requestHostname,
+	technology,
+	experienceUrl
+}) => {
+	let url = `https://playground.mrf.io/simulate?siteUrl=${siteUrl}&requestHostname=${requestHostname}&technology=${technology}&experienceUrl=${experienceUrl}`;
+
+	if (process.env.NODE_ENV === 'production') {
```

Comment:
Its confusing that production environment means using bs-local.

We have different scenarios and i think the current use of NODE_ENV doesnt allow some:
```

test using local appium  OK

test using browserstack with playground KO

test using browserstack-local with playground setting flowcards hostname OK


```

Reply:
I thought `browserstack with playground` would not be a scenario we want to cover. Just `browserstack-local`. Since I did not consider this third scenario I found appropiate to use NODE_ENV `prod` and `dev` here.

So... is `browserstack with playground` a scenario we want to support?

Reply:
I guess yes, its like executing the tests with the production version of flowcards. We will use that to develop the tests actually.

Reply:
I thought we did not want user to spend quota from browserstack.
I will use a different environment variable with 3 scenarios then

Reply:
Its for the flowcards-playground pipeline, its going to be jenkins